guides/doc-Quickstart_Guide/master.adoc

```diff
@@ -0,0 +1,74 @@
+include::common/attributes.adoc[]
+
+:imagesdir: common/images
+:numbered:
+ifdef::foreman-el,foreman-deb,katello[]
+:toc:
+:toc-placement: left
+endif::[]
+
+:quickstart-installation-guide:
+:context: {project-context}
+:mode: connected
+:ProductName: {ProjectServer}
+
+= {project-installation-guide-title}
+
+The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup. It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
+
+Components include the Foreman web UI, Smart Proxy, a Puppet server, and optionally Passenger, TFTP, DNS and DHCP servers. It is configurable and the Puppet modules can be read or run in “no-op” mode to see what changes it will make.
+
+include::common/modules/ref_supported-operating-systems.adoc[]
```

Comment:
Got it now -- I think this is what you meant :)

Reply:
Precisely!